pywr-schema/src/parameters/control_curves.rs

```diff
@@ -106,15 +106,22 @@ impl TryFromV1<ControlCurveInterpolatedParameterV1> for ControlCurveInterpolated
         };
 
         // v1 uses proportional volume for control curves
-        let storage_node = NodeAttrReference {
-            name: v1.storage_node,
-            attribute: Some(NodeAttribute::ProportionalVolume),
+        let storage_metric = if conversion_data.virtual_nodes.contains(&v1.storage_node) {
```

Comment:
Is this going to be dependent on whether the virtual node was converted first or not (e.g. inline parameters)?